engine/table/src/main/java/io/deephaven/engine/table/impl/sources/regioned/RegionedColumnSourceManager.java

```diff
@@ -295,7 +295,7 @@ private WritableRowSet update(final boolean initializing) {
                  * to close() the previous row sets for modified locations. This is not important for current
                  * implementations, since they always allocate new, flat RowSets.
                  */
```

Comment:
This comment might need to be updated since we are no longer using getRowSet here.

Reply:
This was indirectly a very good catch, thanks!